sdk/formrecognizer/Azure.AI.FormRecognizer/tests/Models/FormRecognizerModelFactoryTests.cs

```diff
@@ -355,7 +415,7 @@ public void FormRecognizerModelFactoryCanInstantiateFormField()
             var name = "Cardano";
             var labelData = new FieldData(default, default, default, default);
             var valueData = new FieldData(default, default, default, default);
-            var value = new FieldValue("Hopper");
+            var value = new FieldValue(1);
```

Comment:
We don't have the `FieldValue(string)` internal constructor anymore (it's `FieldValue(string, FieldValueType)` now). Just changed the constructor we're calling, but this doesn't matter for this test.